communication/src/logging.rs

```diff
@@ -20,6 +20,8 @@ pub enum CommunicationEvent {
     Message(MessageEvent),
     /// A state transition.
     State(StateEvent),
+    /// Setup event
+    Setup(CommunicationSetup)
```

Comment:
Are we certain this actually gets constructed anywhere? I didn't see it, but I may have missed it.